ci/tests/vmachine/vmachine_test.py

```diff
@@ -122,6 +125,9 @@ def vms_with_fio_test():
     general.check_prereqs(testcase_number=1,
                           tests_to_run=testsToRun)
 
+    if not DEVSTACK_INSTALLED:
```

Comment:
Does this test need to be skipped without devstack?
Its a vMachine test, shouldnt this be independent of devstack